### ISSUE TITLE
absent lobe of thyroid gland

### ISSUE THREAD

**Issue by anna-anagnostop** [obophenotype/mammalian-phenotype-ontology#4151]
absent lobe of thyroid gland MP:0013937 
Current definition: absence of one or more of the conical shaped sections of the thyroid gland

RGD has no direct annotations; MGI has only one annotation from DMDD ([J:239583](https://www.informatics.jax.org/reference/J:239583))

Consider changing the term label to 'decreased thyroid gland lobe number'
DEF: a reduction in the number of the conical shaped sections of the thyroid gland; in most mammals, the thyroid gland consists of two lobes (left and right) connected by an isthmus portion crossing the upper trachea
SOURCE: PMID:28634271

Comment: The thyroid gland varies greatly in different mammals. In dogs and cats, the gland consists of separate masses connected by an isthmus. In horse, paired lobes are highly dissociated but connected by an insubstantial isthmus. In yet other species, the thyroid has a more compact form and exhibits relatively large median (pyramidal) lobe in addition to lateral lobes. This arrangement is found in pigs and human see https://www.jebmh.com/articles/a-comparative-study-of-thyroid-gland-in-some-mammals.pdf.pdf

Create a new term 'absent thyroid gland lobes' ?

**Comment by anna-anagnostop** [comment]
Changed the term label to 'decreased thyroid gland lobe number' and added a comment. EQ now uses 'decreased amount' and 'lobe of thyroid gland' UBERON:0001118. No need to create a new term for 'absent thyroid gland lobes'.
